### PR TITLE
[feature](profile) add non-zero counter in profile 

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

before
```
                              -  WaitForDependencyTime:  0ns
                                  -  WaitForData0:  1s437ms
                                  -  WaitForData1:  0ns
                                  -  WaitForData10:  0ns
                                  -  WaitForData11:  0ns
                                  -  WaitForData12:  0ns
                                  -  WaitForData13:  0ns
                                  -  WaitForData14:  0ns
                                  -  WaitForData15:  0ns
                                  -  WaitForData16:  0ns
                                  -  WaitForData17:  0ns
                                  -  WaitForData18:  0ns
                                  -  WaitForData19:  0ns
                                  -  WaitForData2:  0ns
                                  -  WaitForData20:  0ns
                                  -  WaitForData21:  13.11us
                                  -  WaitForData22:  0ns
                                  -  WaitForData23:  0ns
                                  -  WaitForData24:  0ns
                                  -  WaitForData25:  0ns
                                  -  WaitForData26:  0ns
                                  -  WaitForData27:  0ns
                                  -  WaitForData28:  0ns
                                  -  WaitForData29:  0ns
                                  -  WaitForData3:  138.392us
```

now

```

                                -  WaitForDependencyTime:  avg  0ns,  max  0ns,  min  0ns
                                    -  WaitForData0:  avg  1s414ms,  max  1s414ms,  min  1s414ms
                                    -  WaitForData12:  avg  66.828us,  max  66.828us,  min  66.828us
                                    -  WaitForData13:  avg  123.439us,  max  123.439us,  min  123.439us
                                    -  WaitForData3:  avg  454ns,  max  454ns,  min  454ns
                                    -  WaitForData34:  avg  8.195us,  max  8.195us,  min  8.195us
                                    -  WaitForData35:  avg  334.533us,  max  334.533us,  min  334.533us
                                    -  WaitForData4:  avg  85.112us,  max  85.112us,  min  85.112us

```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

File: be/src/pipeline/exec/exchange_sink_operator.cpp [hunk 1]
```diff
@@ -197,9 +197,9 @@ Status ExchangeSinkLocalState::init(RuntimeState* state, LocalSinkStateInfo& inf
             if (channel->is_local()) {
                 _local_channels_dependency[dep_id] = channel->get_local_channel_dependency();
                 DCHECK(_local_channels_dependency[dep_id] != nullptr);
-                _wait_channel_timer[dep_id] = ADD_CHILD_TIMER_WITH_LEVEL(
-                        _profile, fmt::format("WaitForLocalExchangeBuffer{}", dep_id), timer_name,
-                        1);
+                _wait_channel_timer[dep_id] = _profile->add_nonzero_counter(
+                        fmt ::format("WaitForLocalExchangeBuffer{}", dep_id), TUnit ::TIME_NS,
```
Comment (HappenLee) [inline be/src/pipeline/exec/exchange_sink_operator.cpp:201]: delele unless space fmt::format

File: be/src/pipeline/exec/exchange_source_operator.cpp [hunk 2]
```diff
@@ -85,8 +85,8 @@ Status ExchangeLocalState::init(RuntimeState* state, LocalStateInfo& info) {
     static const std::string timer_name = "WaitForDependencyTime";
     _wait_for_dependency_timer = ADD_TIMER_WITH_LEVEL(_runtime_profile, timer_name, 1);
     for (size_t i = 0; i < queues.size(); i++) {
-        metrics[i] = ADD_CHILD_TIMER_WITH_LEVEL(_runtime_profile, fmt::format("WaitForData{}", i),
-                                                timer_name, 1);
+        metrics[i] = _runtime_profile->add_nonzero_counter(fmt ::format("WaitForData{}", i),
```
Comment (HappenLee) [inline be/src/pipeline/exec/exchange_source_operator.cpp:88]: same to up

**Comment by Mryange** [comment]
run buildall

**Review by HappenLee — DISMISSED** [review]
LGTM

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**